vampire_survivors/src/highscore.py

```
@@ -0,0 +1,23 @@
+    def __init__(self):
+        self.highscore = 0
+        self.load_highscore()
+
+    def load_highscore(self):
+        try:
+            with open('highscore.json', 'r') as f:
+                self.highscore = json.load(f)
+        except FileNotFoundError:
+            self.highscore = 0
+
+    def save_highscore(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `save_highscore`

Add return type annotation: `None`

(ANN201)